forum/qa-include/pages/account.php

```diff
@@ -326,6 +326,7 @@
 	if ($isblocked) {
 		unset($qa_content['form_profile']['buttons']['save']);
 		$qa_content['error']=qa_lang_html('users/no_permission');
+		$qa_content['http_status']=403;

```

Comment:
A może by tak te status błędów w jakiś słownik wrzucić? Wtedy np. zamiast `403` użyć `$HTTP_STATUS.FORBIDDEN` itd.

Reply:
Słownikiem miałoby być co? Chyba albo jakaś luźna stała, ewentualnie stała w jakiejś klasie. Tylko wtedy trzeba wszędzie posprawdzać, aby była widoczna w danym miejscu, dlatego po prostu wybrałem sobie drogę na skróty :P Statusy HTTP się raczej nie zmienią, no ale można tak niby zrobić dla czytelności.

Reply:
Tymczasowo można taki słownik (obiekt, czy tablicę asocjacyjną) zrobić jako stałą globalną, a kiedyś pozamieniać pozostałe wystąpienia luźnych statusów na coś bardziej odpowiedniego.

Reply:
Ok, zmieniłem na klasę-słownik, Q2A ma jakiś autoloader dla klas i z tego co sprawdziłem to działa wszędzie